tools/install_intel_oneAPI_windows.bat

```diff
@@ -0,0 +1,21 @@
+REM Reference - https://github.com/oneapi-src/oneapi-ci/blob/master/scripts/install_windows.bat
+REM SPDX-FileCopyrightText: 2022 Intel Corporation
+REM
+REM SPDX-License-Identifier: MIT
+
+
+set URL=%1
+set COMPONENTS=%2
+
+
+curl.exe --output %TEMP%\webimage.exe --url %URL% --retry 5 --retry-delay 5
+start /b /wait %TEMP%\webimage.exe -s -x -f webimage_extracted
+del %TEMP%\webimage.exe
+if "%COMPONENTS%"=="" (
+  webimage_extracted\bootstrapper.exe -s --action install --eula=accept -p=NEED_VS2017_INTEGRATION=0 -p=NEED_VS2019_INTEGRATION=0 -p=NEED_VS2022_INTEGRATION=0
```

Comment:
How is it that all the VS integrations are set to 0? Aren't we combining ifx with msvc? Or can you comment why this is switched off?

Reply:
We don't need VS integrations. I have tried both (VS integrations set to 0 and otherwise). It doesn't make a difference. Without these options, the cache size increases. We are using `ifx` for now and `cl` and `ifx` both can work independently with VS integrations set to 0.